netflix-roulette/src/store/actions.ts

```diff
@@ -0,0 +1,67 @@
+import {
+  FetchMoviesError,
+  FetchMoviesErrorPayload,
+  FetchMoviesRequest,
+  FetchMoviesSuccess,
+  FetchMoviesSuccessPayload,
+  SetIsAddMovieDialogVisible,
+  SetIsAddMovieDialogVisiblePayload,
+  FilterMovies,
+  FilterMoviesPayload,
+  SetMovieToEditOrViewPayload,
+  SetMovieToEditOrView,
+  SortMoviesPayload,
+  SortMovies,
+} from "./interfaces";
+import { ActionNames } from "./data";
+
+export const fetchMoviesRequest = (): FetchMoviesRequest => ({
+  type: ActionNames.FETCH_MOVIES_REQUEST,
+});
+
+export const fetchMoviesSuccess = (
+  payload: FetchMoviesSuccessPayload
```

Comment:
You don't necessarily need to wrap each and every action payload into extra layer of `payload` property. Passing properties with more meaningful names could both bring in more _semantics_ and make your code less verbose.